### PR TITLE
Render a new search page if no postcode given

### PR DESCRIPTION
**Pull request by andrewgarner**

### CODE REVIEW

File: app/views/locations/invalid_postcode.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <% content_for(:page_title, "Face-to-face locations near you - Pension Wise") %>
```
Comment (stevenwilkin) [inline app/views/locations/invalid_postcode.html.erb]: Should the title be _Find a face-to-face appointment location..._ like in `search.html.erb`? The `h1`s are the same for both these views.

Comment (andrewgarner) [inline app/views/locations/invalid_postcode.html.erb]: Yes - i'll fix.

**Comment by andrewgarner** [comment]
Title amended @stevenwilkin

**Comment by stevenwilkin** [comment]
:+1:

**Pull request merged**